src/components/Header/Header.module.scss

```diff
@@ -11,21 +11,61 @@
   justify-content: space-between;
   flex-wrap: wrap;
   padding: 3rem 0;
+  gap: 2rem;
```

Comment:
This gap property is very unknown and might confuse people as it's not apparent that there is a `grid` somewhere. Is there a way to have margin instead?

Reply:
There isn't a grid, but [`gap` also works for flexbox](https://caniuse.com/flexbox-gap) which is how I'm using it here. We could do it with margin, but we would have to add margins on each element in the flex container and then negate those margins on the container itself. I think that would end up being a more complex/confusing solution.